src/master/connecting/sdks.md

```diff
@@ -16,11 +16,11 @@ The most well-known and recommended immudb SDK is written in [Golang](https://go
 
 | Language         | Maintainer | Latest immdb version | link | Notes                                                                              |
 |-------------------|---------|------------------|-------------|-----------------------------------------------------------|
-| `go`               | immudb team  | 1.3.2       |     [link](https://pkg.go.dev/github.com/codenotary/immudb/pkg/client)  |                                   |
-| `python`               | immudb team  | 1.3.2       |  [link](https://github.com/codenotary/immudb-py) |                                     |
-| `JAVA`               | immudb team  | 1.3.2       |   [link](https://github.com/codenotary/immudb4j)  |                                      |
+| `go`               | immudb team  | 1.4.0       |     [link](https://pkg.go.dev/github.com/codenotary/immudb/pkg/client)  |                                   |
+| `python`               | immudb team  | 1.4.0       |  [link](https://github.com/codenotary/immudb-py) |                                     |
+| `JAVA`               | immudb team  | 1.4.0       |   [link](https://github.com/codenotary/immudb4j)  |                                      |
 | `NODE`               | immudb team | 1.2.1       |   [link](https://github.com/codenotary/immudb-node) | Verification is not working                 |
-| `ruby`               | Community ([Ankane](https://github.com/ankane))  | 1.3.2       |   [link](https://github.com/ankane/immudb-ruby) |                 |
+| `ruby`               | Community ([Ankane](https://github.com/ankane))  | 1.4.0       |   [link](https://github.com/ankane/immudb-ruby) |                 |
```

Comment:
Should we claim that it is supported before ruby SDK is updated ?

Honestly I think that this table is not correct, it should contain something like `most recent SDK version` and `minimum required immudb version`

Reply:
It is even possible that we may need a table for each SDK, listing the immudb versions that work with that SDK version